packages/ui-components/src/Avatar/Avatar.module.scss

```diff
@@ -0,0 +1,98 @@
+@import "../styles/tokens.scss";
+
+@mixin selectable-border($n) {
+  box-sizing: border-box;
+  border: 2px solid $n;
+}
```

Comment:
This seems like it would be a good candidate to be added to our tokens, probably with a default color value.  It might not be the right time to do so now (i.e. when we have more use cases) but let's add a TODO comment above this to call it out.

Also the let's make var name something descriptive; for example `$color`... `$n` has connotations with mathematics as relating to an integer count value so it makes it slightly (though not fatally) confusing.